Development

josephcyang4459 opened the pull request:
Please look at the changes done (specifically the script changes. scene and asset changes are too hard to look at).

There's quite a bit of stuff happening here. The game is mostly operational after the changes we've made. I haven't came across bugs that will cause a hard crash.

Here are a list of changes for quick reference.
**Changes**
- fixed missing character scene stat increases (particularly for intoxication)
- removed and replaced all references of charisma with skill
- fixed a few typos with a few of the flags in the drinking repeat scenes
- Refactored a few blocks of code
- Reformatted a few methods so that curly brackets are on same line as method header or conditional statement
- Replaced all references of "players" in numerous scripts with references to a new scriptable object "characters" instead
- time display logic refixed

JessFletcher reviewed (approved):
I have implemented the File Changes you have made to the Cutscene Scripts in the Shared Google Drive. I have also updated my Visual Studio to have the same new line behavior.

josephcyang4459 commented:
> I have implemented the File Changes you have made to the Cutscene Scripts in the Shared Google Drive. I have also updated my Visual Studio to have the same new line behavior.

Hell yar brother.
